core/src/main/resources/i18n/displayStrings.properties

```
@@ -1137,8 +1137,10 @@ support.closeTicket=Close ticket
 support.attachments=Attachments:
 support.savedInMailbox=Message saved in receiver's mailbox
 support.arrived=Message arrived at receiver
+support.transient=Message is on its way to receiver
 support.acknowledged=Message arrival confirmed by receiver
 support.error=Receiver could not process message. Error: {0}
+support.errorTimeout=Timed-out waiting for ACK, try again
```

Is this suggestion to "try again" directed at the user?

If so, something like this might be more clear:

```suggestion
support.errorTimeout=timed out. Try sending message again.
```